src/api/mod.rs

```diff
@@ -672,6 +692,18 @@ impl<T: Pixel> IntoFrame<T> for (Arc<Frame<T>>, Option<FrameParameters>) {
 }
 
 impl<T: Pixel> Context<T> {
+  /// Allocates and returns a new frame.
+  ///
+  /// # Examples
+  ///
+  /// ```
+  /// use rav1e::prelude::*;
+  ///
+  /// let cfg = Config::default();
+  /// let ctx: Context<u8> = cfg.new_context();
+  /// let frame = ctx.new_frame();
+  /// ```
+  #[inline]
```

Comment:
Why does a documentation PR contain functional (well, to the compiler) changes?

Reply:
I should split the inlines into a separate PR then.

Reply:
OK. They're not wrong, I was just wondering why they're in this PR.